pkg/cmd/server/admin/create_clientcert.go

```diff
@@ -62,3 +65,9 @@ func (o CreateClientCertOptions) CreateClientCert() (*crypto.TLSCertificateConfi
 		return signerCert.EnsureClientCertificate(o.CertFile, o.KeyFile, userInfo)
 	}
 }
+
+func (o CreateClientCertOptions) Printf(tmpl string, args ...interface{}) {
```

Comment:
Rather than have a printing function, it probably cleaner to have a `GetOut()` method that always returns a valid `io.Writer`.  Either what's been on the object or `ioutil.Discard`.  That's more generally helpful and will prevent people from doing crazy stuff in here.